src/peft/tuners/lora/config.py

```diff
@@ -117,6 +118,25 @@ class LoraConfig(PeftConfig):
             )
         },
     )
+    bits_pattern: Optional[dict] = field(
+        default_factory=dict,
+        metadata={
+            "help": (
+                "The mapping from layer names or regexp expression to ranks which are different from the default bits specified by `bits`. "
+                "For example, `{model.decoder.layers.0.encoder_attn.k_proj: 2`}"
+            )
+        },
+    )
+    loftq_bits: Optional[int] = field(default=4, metadata={"help": "Quantization bits for LoftQ"})
```

Comment:
The help for this option, as well as for `loftq_iter` is a bit short. Could you please elaborate, e.g. what are valid options (IIUC only 4 and 8), or at least provide a reference for users to better understand the options?